### PR TITLE
[zh] fix wrong anchor link

### PR DESCRIPTION
**Pull request by yaowenqiang**
fix one wrong anchor link in content/zh/docs/tasks/access-application-cluster/access-cluster.md

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@yaowenqiang
Pls sign the CLA in order for the PR to be reviewed.

**Comment by yaowenqiang** [comment]
> @yaowenqiang
> Pls sign the CLA in order for the PR to be reviewed.

Done

File: content/zh/docs/tasks/access-application-cluster/access-cluster.md [hunk 1]
```diff
@@ -371,7 +371,7 @@ You have several options for connecting to nodes, pods and services from outside
       如果服务不能够安全地暴露到互联网，或者服务不能获得节点 IP 端口的访问权限，或者是为了 debug，那么请使用此选项。
     - 代理可能会给一些 web 应用带来问题。
     - 只适用于 HTTP/HTTPS。
-    - 更多详细信息在 [这里]。
+    - 更多详细信息在 [这里](#发现内建服务)。
```
Comment (tanjunchen) [inline content/zh/docs/tasks/access-application-cluster/access-cluster.md:374]: Described [here](#manually-constructing-apiserver-proxy-urls)

it seems that is not right.

Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/access-cluster.md:374]: please use english anchor name when possible, although Chinese anchor names work too.

**Comment by yaowenqiang** [comment]
the original english anchor name ihas been translated into Chinese, so to link to the anchor, the anchor name should also be Chinese

**Comment by tengqm** [comment]
@yaowenqiang You can add an explicit anchor using the following syntax:

```
## 介绍  {#introduction}
```

**Comment by yaowenqiang** [comment]
@tengqm tks, I will close this pr and send a new one.

**Pull request closed without merging**